presto-docs/src/main/sphinx/sql/show-roles.rst

```diff
@@ -0,0 +1,15 @@
+==========
+SHOW ROLES
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW ROLES [ FROM catalog ]
```

Comment:
I did the same sneaky thing that FB does, allows IN in the grammar but only documents FROM. This is consistent through all SHOW commands.